index.js

```diff
@@ -114,6 +114,16 @@ function SendStream(req, path, options) {
     ? normalizeList(opts.extensions, 'extensions option')
     : []
 
+  if (Array.isArray(opts.precompressed)) {
+    this._precompressionFormats = opts.precompressed
+  } else if (opts.precompressed) {
+    this._precompressionFormats = [{encoding: 'br', extension:'.br'}, {encoding:'gzip', extension:'.gz'}]
```

Comment:
Content-encoding is not a file extension, otherwise it would be .gzip, not .gz. Can you link me to the place where the file extension is specified?